<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DATwoValues</class>
 <widget class="QDialog" name="DATwoValues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.990000000005239</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interpolator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DATwoValues</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DATwoValues</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
